### PR TITLE
qt5pas: update to 2.2.0

### PR DESCRIPTION
**Pull request by d33vliter**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

### CODE REVIEW

**Comment by paper42** [comment]
one commit per package

File: srcpkgs/qt5pas/template [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 makedepends="qt5-devel qt5-x11extras-devel"
 short_desc="Qt5 interface bindings for Pascal"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="d33vliter <d33vliter@protonmail.com>"
 license="LGPL-3.0-or-later"
-homepage="https://svn.freepascal.org/svn/lazarus/trunk/lcl/interfaces/qt5/cbindings"
-distfiles="${DEBIAN_SITE}/main/libq/libqtpas/libqtpas_${version}.orig.tar.xz"
-checksum=fcc5f695d60d5f72bf501b2fa779e8a1a2636c8d991270b5b3fef3d0e990365c
+homepage="https://gitlab.com/freepascal.org/lazarus/lazarus/-/tree/main/lcl/interfaces/qt5/cbindings"
+distfiles="http://deb.debian.org/debian/pool/main/libq/libqtpas/libqtpas_2.6+${version}+dfsg1.orig.tar.xz"
```
Comment (paper42) [inline srcpkgs/qt5pas/template:12]: use $DEBIAN_SITE

File: srcpkgs/qt5pas/template [hunk 2]
```diff
@@ -1,16 +1,17 @@
 # Template file for 'qt5pas'
 pkgname=qt5pas
-version=2.6~beta
+version=2.2.0
```
Comment (paper42) [inline srcpkgs/qt5pas/template:3]: I am pretty sure this will be considered a downgrade by xbps, so people won't get it

**Comment by paper42** [comment]
This is still 5 commits, not 1, please squash them

File: srcpkgs/qt5pas/template (hunk 2 above)
Comment (iadupuy) [reply srcpkgs/qt5pas/template:3]: I don't understand that much, but the current version is more outdated, I didn't understand why they put 2.6 beta when it wasn't like that

**Comment by paper42** [comment]
please don't close this, instead, force push to the branch

**Pull request closed without merging**